src/components/People.tsx

```diff
@@ -0,0 +1,319 @@
+import { useState, useEffect } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { PersonInfo } from './PersonInfo';
+import { PeopleFilters } from './PeopleFilters';
+import { SortTypes } from '../types/SortTypes';
+import { Centuries } from '../types/Centuries';
+import { SearchLink } from './SearchLink';
+import { Sex } from '../types/Sex';
+
+function sortAndFilter(
+  people: Person[],
+  sortColumn: SortTypes,
+  sortIsReverse: boolean,
+  filterSex: string,
+  filterCentury: string[],
+  filterQuery: string,
```

Comment:
Instead of storing this values in state and passing them here, they could be accessed with useSearchParams hook and then e.g. searchParams.get('name')